charts/barman/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: barman
+type: application
+description: Barman
```

Comment:
Please change this to a full sentence since it is used as the first part of the README generated by `helm-docs`.